### PR TITLE
created get request for get slider img

### PR DESCRIPTION
**Pull request by Ksenaaa**

### CODE REVIEW

File: src/components/Slider/Slider.tsx [hunk 1]
```diff
@@ -1,29 +1,41 @@
...
+        setNumberPic(preState => pictures.length-1 === preState ? 0 : preState + 1)
     }
 
     const previousPicture = () => {
-        setNumberPic(preState => preState === 0 ? sliderPics.length-1 : preState - 1)
+        setNumberPic(preState => preState === 0 ? pictures.length-1 : preState - 1)
     }
 
     const findItem = (num: number) => {
         const child = refContainer?.current as HTMLDivElement
```
Comment (vell28) [inline src/components/Slider/Slider.tsx:38]: if (!refContainer?.current) return

File: src/components/Slider/Slider.tsx [hunk 2]
```diff
@@ -1,29 +1,41 @@
...
 import { SliderItem } from './SliderItem'
+import { useHttp } from 'hooks/http.hooks'
+import { IImgSlider } from 'interface/IImgSlider'
 
 import styles from './Slider.module.scss'
-import { sliderPics } from 'constants/sliderPics'
 
 export const Slider = () => {
+    const [pictures, setPictures] = useState<IImgSlider[]>([])
     const [numberPic, setNumberPic] = useState(0)
```
Comment (vell28) [inline src/components/Slider/Slider.tsx]: activePicture

**Review by vell28 — APPROVED** [review]

File: src/components/Slider/Slider.tsx (hunk 2 above)
Comment (Ksenaaa) [reply src/components/Slider/Slider.tsx]: fixed

File: src/components/Slider/Slider.tsx (hunk 1 above)
Comment (Ksenaaa) [reply src/components/Slider/Slider.tsx:38]: fixed

**Pull request merged**